tests/search/nearest_neighbor/mixed/test.sd

```diff
@@ -0,0 +1,62 @@
+schema test {
```

Comment:
Please add copyright. The same applies for test.sd used for the dense case.

Reply:
Done.